### ISSUE TITLE
Invalid position given by 'no-multiple-empty-lines'

### ISSUE THREAD

**Issue by atardadi** [AtomLinter/linter-eslint#830]
ESLint returned a point that did not exist in the document being edited.
Rule: `no-multiple-empty-lines`
Requested range: 121:1

Debug information:
```json
{
  "atomVersion": "1.14.3",
  "linterEslintVersion": "8.1.2",
  "linterEslintConfig": {
    "lintHtmlFiles": false,
    "useGlobalEslint": false,
    "showRuleIdInMessage": true,
    "disableWhenNoEslintConfig": true,
    "eslintrcPath": "",
    "globalNodePath": "",
    "advancedLocalNodeModules": "",
    "eslintRulesDir": "",
    "disableEslintIgnore": false,
    "disableFSCache": false,
    "fixOnSave": false,
    "scopes": [
      "source.js",
      "source.jsx",
      "source.js.jsx",
      "source.babel",
      "source.js-semantic"
    ],
    "rulesToSilenceWhileTyping": [],
    "rulesToDisableWhileFixing": []
  },
  "eslintVersion": "2.13.1",
  "hoursSinceRestart": 6.3,
  "platform": "darwin",
  "eslintType": "local project",
  "eslintPath": "/Users/atardadi/git/medalliadigitalchatbot/facebook-chatbot/node_modules/eslint"
}
```

**Comment by Arcanemagus** [comment]
Please update your version of ESLint, that rule had a bug that was fixed in v3.5.0.

**Comment by Arcanemagus** [comment]
Closing due to a lack of response, if you are still having problems after updating ESLint please report back, thanks!
